src/ripple/nodestore/impl/varint.h

```diff
@@ -28,7 +28,7 @@ namespace beast {
 namespace nudb {
```

Comment:
Nik beat me to it...
The dependency on `nudb/detail/stream.h` (and `field.h`) looks a bit questionable now.
Should they be levelised out of nudb (they only appear to be used there and nodestore)?